### ISSUE TITLE
Add Sponsors to Sponsors section

### ISSUE THREAD

**Issue by GovInTrenches** [codeforamerica/brigade#453]
Need to add Microsoft and Knight to list of Brigade sponsors

**Comment by tdooner** [comment]
@GovInTrenches Is this still relevant? Is this still the full list of sponsors?

**Comment by GovInTrenches** [comment]
Yes to Knight, no to Microsoft. We should also be sure to add Sloan Foundation and CA Technilogies if they're not there already
